libraries/AP_Scripting/drivers/mount-djirs2-driver.lua

```diff
@@ -653,12 +656,27 @@ function parse_byte(b)
           end
 
           -- parse attitude reply message
-          if (expected_reply == REPLY_TYPE.ATTITUDE) and (parse_length >= 20) then
-            local ret_code = parse_buff[13]
+          if (expected_reply == REPLY_TYPE.ATTITUDE) and (parse_length >= ATTITUDE_PACKET_LEN.LEGACY) then
+            -- default to legacy format but also handle latest format
+            local ret_code_field = 13
+            local yaw_field = 15
+            local pitch_field = 17
+            local roll_field = 19
```

Comment:
Would like to see a comment here that this field ordering contradicts the spec but is how an actual gimbal of version XX (whichever version yours is) is observed to behave.